bundles/org.openhab.core/src/test/java/org/openhab/core/util/ColorUtilTest.java

```diff
@@ -435,7 +435,7 @@ private static Stream<Arguments> allXY() {
      */
     private static Stream<Arguments> allRGBW() {
         List<Arguments> result = new ArrayList<>();
-        final double step = 5.0;
+        final double step = 17.0;
```

Comment:
Do you want to hit the upper boundary 100? Then a factor of 100 would be a better choice.
With step size 17, the loop will end at 85.

Reply:
I know. I was trying to set a step that tests intersticial values between the obvious round numbers. So I was thinking of using primes. But perhaps a factor of 100 would indeed be better..